.github/workflows/publish.yml

```diff
@@ -58,12 +58,31 @@ jobs:
         run: |
           echo "ARCH=${{ matrix.arch }}" >> "$GITHUB_ENV"
           echo "TAG=$(echo "$REF" | sed -e 's#^.*/##' -e 's#master#snapshot#' -e 's#main#snapshot#')" >> "$GITHUB_ENV"
-      - name: Login to Docker Hub
+      - name: Login to Docker Hub (build)
+        uses: docker/login-action@v3
```

Comment:
Don't we want to have ` if: ${{ github.event.repository.full_name }} == 'lf-edge/eve'` here as well?

Reply:
It makes sense. I just decided not to change the original logic.

Reply:
Added the condition.